make.js

```diff
@@ -86,6 +86,20 @@ target.test = function() {
     run('mocha test/tests');
 }
 
+target.testNoBuild = function() {
+    pushd('test')
+    run('npm install ../_build');
+    popd();
+
+    console.log("-------Unit Tests-------");
+    run('tsc -p ./test/units');
+    run('mocha test/units');
+
+    console.log("-------Other Tests-------");
```

Comment:
What are "other" tests?

Reply:
Non-nock tests. I don't really know what the best name for them is, the only difference here is that they actually make http calls. They're not really integration tests since they're still only testing individual components, but they also are a little more involved/flaky than the units.

Reply:
Integration is close enough I think.

Reply:
Ok, should be good to go